.github/workflows/prerelease_linux.yml

```diff
@@ -0,0 +1,114 @@
+name: Prerelease Linux pipeline
+
+on:
+  release:
+    types:
+      - prereleased
+    tags:
+      - 'v*'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
is it upper or lower case or? https://github.com/newrelic/infrastructure-agent/pull/298#diff-75a012a22e2ed993e3c1cf7e354013b0f656030461340731bd3837aea269fea6R50

Reply:
updated to uppercase in all places